docs/running-on-kind.md

```diff
@@ -0,0 +1,37 @@
+# Running on Kind
+
+This guide walks you through how to run NGINX Kubernetes Gateway on a [Kind](https://kind.sigs.k8s.io/) cluster.
+
+## Prerequisites
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+- [kind](https://kind.sigs.k8s.io/)
+
+## Prepare Cluster
+
+Create a cluster with Kind. You can follow their [instructions](https://kind.sigs.k8s.io/docs/user/quick-start/#creating-a-cluster), or run the following make command at the root of the repository:
```

Comment:
```suggestion
Create a cluster with `kind`. You can follow their [instructions](https://kind.sigs.k8s.io/docs/user/quick-start/#creating-a-cluster), or run the following make command at the root of the repository:
```

I noticed that `kind` is never capitalized in their docs, not sure if we want to follow that convention and maybe just highlight it?